### ISSUE TITLE
[Suggestion] changing of the covenant job names

### ISSUE THREAD

**Issue by Joe44445** [HaloSpaceStation/HaloSpaceStation13#2508]
instead of calling the Covenant jobs by their lore names (i.e Sangheilli, unggoy, Kig Yar), call them by the UNSC defined names to make them more recognisable to people new to the server who may have little halo lore knowledge allowing them to play as the covenant right off the bat. it will hopefully make the covenant roles look more appealing to new players as it will be recognisable to anyone who has played the halo games but not really delved much further into halo lore to know their actual species names. Should hopefully help allveiate covenant population problems.

**Comment by Sgtwok** [comment]
Covenant here, nah.

**Comment by loafe** [comment]
The average unga bunga ss13 player doesn't know the deep lore of the Halo universe, they just know what they are called from the games. Calling them their name from the Halo games might help people understand what they are actually signing up for.

Likewise, when new players are joining the server, they likely do not know how to change their race, and thus can't actually become a covenant because the default race in ss13 is human. A system which makes it easier for new players to become Covenant (Like joining as Xenos in CM) would be better for improving Covie pop.

**Comment by thog1** [comment]
After browsing through elite quotes for halo three I noticed that not once are the brutes called "Jiralhananenae" or whatever it is. Obviously this is to cater for the audience  and that's exactly what the occupation menu is for - the audience or in HS13's case, the players. The covenant arent gonna be walking around with ID tags saying their species names are they. Its simply for accessibility for new players. Were not changing the species name, merely the occupation title. 


Source - https://www.halopedia.org/Sangheili/Quotes#:~:text=%22The%20Demon%20is%20here!%22,the%20Master%20Chief%20is%20hiding.

**Comment by CommanderXor** [comment]
The Jiralhanae are referenced in the series as that, including ingame and on official Halo pages. Literally the same source you linked has them referred to as Jiralhanae.

https://gyazo.com/5c94fc68486dd6caa9c482a8224662d6
https://gyazo.com/6be8ad2106f7e8d699f0dadd3315c329

Including official Halo websites.

https://www.halowaypoint.com/en-us/universe/species/jiralhanae

Personally I'm against the idea of changing them to the UNSC designations just because they do infact wear ID tags ingame and if I see 'elite major' or 'elite shipmaster' I'm yeeting myself down a flight of stairs.

**Comment by thog1** [comment]
Yeah my bad for missing that. I'd really like to come to a compromise to increase accessibility to new players though. Perhaps keep Whitelisted titles the same and change the unwhitelisted ones. Also what are some reasons for NOT changing it? I understand "lore" and all but please remember we are not a book.

**Comment by caelaislinn** [comment]
I made sure to always use the Covenant species names when doing Covenant jobs, but it's a valid point about accessibility. A nice compromise would be to use the human names (brute, elite etc) on the job screen, but the lore names ingame

**Comment by thog1** [comment]
I feel like we should be weighing this issue on the principle of whether or not it will help or hinder the progress of this server, not focused so much on what the old guard are comfortable with, but what will actively aid in growth of this server. I understand why ALOT of the covenant would say no if this was a drastic change but I dont think it is. We're staying close to the lore, covenant species being called by their UNSC designations by other covenant members is not unheard of.

But yeah if caels idea works I'm down with that.

**Comment by BDpuffy420** [comment]
This is stupid, the average greentide doesn’t even know what halo is. We get ahelps like that all the time so using the unsc designation is pointless. Unga dunga wanna shoot xeno is basically it

**Comment by BDpuffy420** [comment]
Like if new person shows up they aren’t going to know what an elite is either and STILL ahelp “what’s halo and what do I do”. Dev focus should be elsewhere rather than this

**Comment by ghost** [comment]
Covenant here, ya.
